examples/features/standard/queue/src/main/java/org/apache/activemq/artemis/jms/example/QueueExample.java

```diff
@@ -37,7 +37,7 @@ public static void main(final String[] args) throws Exception {
          // Step 1. Create an initial context to perform the JNDI lookup.
          initialContext = new InitialContext();
 
-         // Step 2. Perfom a lookup on the queue
+         // Step 2. Perform a lookup on the queue
          Queue queue = (Queue) initialContext.lookup("queue/exampleQueue");
```

Comment:
you know although I would prefer users to not use JNDI. either session.createQueue() or direct create the queue.

there are so many examples that we did not have time to update all of them.. even the create connection factory is preferred to use an URI.